strongbox-security/strongbox-user-management/src/main/resources/etc/conf/strongbox-authorization.yaml

```diff
@@ -1,15 +1,157 @@
+# Following privilegies avaliable:  
+#  - CONFIGURATION_UPLOAD
+#  - CONFIGURATION_VIEW
+#  - CONFIGURATION_SET_INSTANCE_NAME
+#  - CONFIGURATION_VIEW_INSTANCE_NAME
+#  - CONFIGURATION_SET_BASE_URL
+#  - CONFIGURATION_VIEW_BASE_URL
+#  - CONFIGURATION_SET_PORT
+#  - CONFIGURATION_VIEW_PORT
+#  - CONFIGURATION_SET_GLOBAL_PROXY_CFG
+#  - CONFIGURATION_VIEW_GLOBAL_PROXY_CFG
+#  - CONFIGURATION_ADD_UPDATE_STORAGE
+#  - CONFIGURATION_VIEW_STORAGE_CONFIGURATION
+#  - CONFIGURATION_DELETE_STORAGE_CONFIGURATION
+#  - CONFIGURATION_ADD_UPDATE_REPOSITORY
+#  - CONFIGURATION_VIEW_REPOSITORY
+#  - CONFIGURATION_DELETE_REPOSITORY
+#  - CONFIGURATION_ADD_LOGGER
+#  - CONFIGURATION_UPDATE_LOGGER
+#  - CONFIGURATION_DELETE_LOGGER
+#  - CONFIGURATION_RETRIEVE_LOG
+#  - CONFIGURATION_RETRIEVE_LOGBACK_CFG
+#  - CONFIGURATION_UPLOAD_LOGBACK_CFG
+#  - MANAGEMENT_REBUILD_METADATA
+#  - MANAGEMENT_DELETE_METADATA
+#  - MANAGEMENT_REBUILD_INDEXES
+#  - ADMIN_CREATE_REPO
+#  - ADMIN_UPDATE_REPO
+#  - ADMIN_DELETE_REPO
+#  - ADMIN_LIST_REPO
+#  - CREATE_USER
+#  - UPDATE_USER
+#  - VIEW_USER
+#  - IMPERSONATE_USER
+#  - DELETE_USER
+#  - ARTIFACTS_DEPLOY
+#  - ARTIFACTS_DELETE
+#  - ARTIFACTS_VIEW
+#  - ARTIFACTS_RESOLVE
+#  - ARTIFACTS_COPY
+#  - SEARCH_ARTIFACTS
+#  - MANAGEMENT_DELETE_ALL_TRASHES
+#  - MANAGEMENT_DELETE_TRASH
+#  - MANAGEMENT_UNDELETE_ALL_TRASHES
+#  - MANAGEMENT_UNDELETE_TRASH
+#  - VIEW_OWN_TOKEN
+#  - VIEW_ANY_TOKEN
+#  - VIEW_LOGS
+#  - CONFIGURE_LOGS
+#  - RSS_FEED
+#  - UI_LOGIN
+#  - UI_BROWSE
```

Comment:
I assume these commented out privileges can now be removed from the configuration file?

Reply:
dropped